test/module/shared_model/backend_proto/common_objects/proto_common_objects_factory_test.cpp

```diff
@@ -27,6 +27,30 @@ class PeerTest : public ProtoFixture {
   crypto::PublicKey valid_pubkey =
       crypto::DefaultCryptoAlgorithmType::generateKeypair().publicKey();
   std::string invalid_address = "127.0.0.1";
+  std::string example_tls_certificate = R"(
```

Comment:
Not sure that a real certificate is required here, because the format is not validated.